### PR TITLE
update protos

### PR DESCRIPTION
**Pull request by FabianTerhorst**

### CODE REVIEW

**Comment by FabianTerhorst** [comment]
request hash is now a uint64 instead of a int64

File: README.md [hunk 1]
```diff
@@ -171,5 +171,6 @@ You can't. The Google Identity Platform uses the SHA1 fingerprint and package na
   - @mjmfighter
   - @vmarchaud
   - @langerhans
+  - @fabianterhorst
```
Comment (AbandonedCart) [inline README.md:174]: Should I submit a separate PR for it, or did you want to add me to this, too?

Comment (FabianTerhorst) [inline README.md:174]: i added you

Comment (AbandonedCart) [inline README.md:174]: Thanks
